airflow/api_fastapi/execution_api/datamodels/taskinstance.py

```diff
@@ -157,6 +157,13 @@ class TIRescheduleStatePayload(StrictBaseModel):
     end_date: UtcDateTime
 
 
+class TISkippedDownstreamTasksStatePayload(StrictBaseModel):
+    """Schema for updating downstream tasks TaskInstance to a skipped state."""
+
+    # FIXME: Should be Union[list[str], list[tuple[str, int]]]
+    tasks: list
```

Comment:
Struggled here with exact typing, the `_generated.py` caused issues.
I guess that we could handle it later  - maybe we could even make it work only with `list[tuple[str ,int]`, but we need to ensure backward compatibility before that.